test/verify/check-apps

```diff
@@ -103,12 +103,20 @@ class TestApps(packagelib.PackageCase):
         b.wait_visible(".pf-v5-c-empty-state")
 
         self.createAppStreamPackage("app-1", "1.0", "1")
+        self.createAppStreamPackage("app-2", "1.0", "1", install=True)
         self.createAppStreamRepoPackage()
 
+        b.enter_page('/apps')
```

Comment:
This is a no-op. This just changes the frame, but `login_and_go` on line 102 already switched to the /apps frame.

Reply:
Right, but I am only evaluating things once, so I need a full refresh instead?

Reply:
What are you actually trying to do? Cockpit pages ought to react to system changes automatically. And your new test seems to work just fine. I don't see anything missing?

Reply:
I moved the `read_os_release` from `refresh` to `useInit(...)` and collected the values, as I don't think these can change dynamically while you are using that page, unless you want me to move it back?

Reply:
Ah yes, then you need to first do `b.reload()` and then `b.enter_page('/apps')` indeed. Just the latter is a no-op.

Reply:
Yep, that explains it, thanks! Works locally for me now.